source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,90 @@
+
+## Entities
+
+Because many devices do more than one thing, things in Home Assistant are controlled using **entities**. An entity is a single function or property of a device, like a single sensor, light, or switch.
+
+A device can provide multiple entities. For example, a device called “Sam’s Phone” might have entities like location, battery level, and step count. Some devices like lightbulbs only have a single entity, and devices like hubs may have no entities at all.
+
+To get entities to appear in Home Assistant, you must define them in your [`configuration.yaml` file](/docs/configuration) in this format: `entity_id: domain.object_id`. The **domain** is a built-in category of entities that you manage in a similar way. You can get the object ID by ________?
+
+Some entities are useful for controlling things in your home:
+
+- **[Switches](/integrations/switch)** let you check if something is switched on or off
+- **[Sensors](/integrations/sensor)** can be a wide range of things that watch for changes in the environment, like motion sensors and temperature sensors
+- **[Binary sensors](/integrations/device_tracker)** detect something that is either on or off
+
+Other entities are there to respond to Home Assistant:
+
+- **[Lights](/integrations/light)** can turn on and off, and sometimes dim and change color
+- **[Covers](/integrations/cover)** are things that can move up and down, like garage doors and smart blinds
+
+Entities can be grouped into **areas**, which roughly represent a room in your house. Common areas are “kitchen”, “office”, “basement” and “yard”.
+
+You can reference multiple entities at one time by putting them into a **[group](/integrations/group)**
+
+_Advanced users can write [scripts](/docs/scripts), special entities that run custom code when they’re triggered_
+
+## Automation
+
+**[Automations](/docs/automation)** let you go further by telling Home Assistant to automatically do things without you having to hit a button yourself. For example, you could write an automation that says “turn on the lights when it gets dark if I’m home”.
+
+An automation has three main parts:
+
+- **[Triggers](/docs/automation/trigger)** - something that starts the automation, like the sun going down or a motion sensor noticing movement
+- **[Conditions](/docs/automation/condition)** - let you decide whether to complete the automation. For example: "only if someone is home".
+- **[Action](/docs/automation/action)** - what the automation does, like "turn on lights"
+
+**[Templates](/docs/automation/templating)** let advanced users use variables and logic in their automations
+
+## State
+
+Home Assistant remembers the **[state](/docs/configuration/state_object)** of each entity. Think of state as in "what state is this in?" For example, a light could have two states: ‘on’ and ‘off’, and a garage door could be in one of two states: ‘open’ or ‘closed’.
```

Comment:
This leads me to believe that mutliple states are stored at one time.  Possibly change
```
For example, a light could have two states: ‘on’ and ‘off’
```
to:
```
For example, a light could have one of two states: ‘on’ and ‘off’
```

Reply:
love it, will fix 👍